datacenter/dtr/2.2/guides/admin/configure/external-storage/index.md

```diff
@@ -6,116 +6,72 @@ title: Configure DTR image storage
 ---
 
 After installing Docker Trusted Registry, one of your first tasks is to
```

Comment:
We should also [include this diagram](https://github.com/docker/docker.github.io/blob/master/datacenter/dtr/2.2/guides/images/architecture-3.svg) right at be beginning, to make it obvious what we're talking about.

Diagrams can be even better than TL;DR. My eyes will immediately jump to the diagram and won't have to spend  several milliseconds scanning for a keyword that catches my attention.